screens/activeRoutes/ActiveRoutes.tsx

```diff
@@ -1,37 +1,35 @@
 import { Box, Flex, Spinner, useColorModeValue } from 'native-base';
-import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
 import RouteContainer from '../../components/route/SearchableRouteContainer';
-import { getActiveRoutesCursor } from '../../xplat/api';
-import { Route } from '../../xplat/types/route';
+import {
+  ACTIVE_ROUTES_CACHE_KEY,
+  fetchActiveRoutes,
+} from '../../utils/queries';
 
 const ActiveRoutes = () => {
-  const [routes, setRoutes] = useState<Route[]>([]);
-  const [isLoading, setIsLoading] = useState<boolean>(true);
-
   const baseBgColor = useColorModeValue('lightMode.base', 'darkMode.base');
 
-  useEffect(() => {
-    const getData = async () => {
-      // There will never be many active routes, so pull them all for searchability
-      const activeRoutesCursor = getActiveRoutesCursor();
-      const newRoutes: Route[] = [];
-      while (await activeRoutesCursor.hasNext()) {
-        newRoutes.push((await activeRoutesCursor.pollNext())!);
-      }
-      setRoutes(newRoutes);
-      setIsLoading(false);
-    };
+  const { isLoading, isError, data, error } = useQuery(
+    ACTIVE_ROUTES_CACHE_KEY,
```

Comment:
I was JUST THINKING pretty much exactly this. thanks again, react query!
However, we should crank up the cachetime and staletime for this, to like 30 minutes. What do you think?

Reply:
Or honestly even longer, like I think it could seriously be like 2 hours, cuz active routes can't be edited and cases where staff publish routes *during the climbing day* are going to be exceedingly rare

Reply:
Updated! Good idea.